src/index.html

```diff
@@ -1,17 +1,164 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Hello</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
   </body>
+  <main>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" class="form-field">
```

Comment:
if the names of attributes are too long - move each of them to a new line.
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post" 
      class="form-field"
    >
```